quiz/web/tsconfig.json

```diff
@@ -17,7 +17,9 @@
         "jsx": "react-jsx",
         "noUnusedLocals": true,
         "noUnusedParameters": true,
-        "types": ["node"]
+        "types": ["node"],
+        "strictNullChecks": true,
+        "strictFunctionTypes": false
```

Comment:
Hmmm co to za wyłączanie type checków 😅

Reply:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/90700152/223877341-98dddc10-27fb-47d0-8011-0009866bc58b.png">

https://github.com/jquense/yup#typescript-configuration